prep-css-grid-classes/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Grid Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+/* stylelint-disable block-no-empty */
+body {
+  font-family: sans-serif;
+}
+
+h1 {
+  text-align: center;
+}
+
+h3 {
+  text-align: center;
+}
+
+p {
+  text-align: left;
+}
+
+img {
+  width: 100%;
+  height: 450px;
+  object-fit: contain;
+}
+
+div {
+  text-align: left;
+}
+
+.container {
+  max-width: 992 px;
```

Comment:
Your `max-width` needs to have the `px` touching the `992`. Like `992px`.

Reply:
Oh woops! I'll correct it one second.